src/Components/Admin/TranscriptsView/index.js

```diff
@@ -48,8 +48,8 @@ const TranscriptsView = props => {
         status:  tr.status,
         created:  tr.created,
         updated:  tr.updated,
-        duration:  upload.duration,
-        size:upload.size,
+        duration:  upload ? upload.duration : 0,
+        size: upload ? upload.size : 0,
```

Comment:
Jw what edge case this is handling? Is this if there is a transcript uploaded but no media is associated with it? Does the paper edit functionality work in that case, or should this be handled as an error on line 38 `await Uploads.getItem(tr.id)`?

Reply:
actually you're right, this isn't an edge case that should be covered. Logically it shouldn't happen - it's probably my broken PR that got it in this state 🤔